vaults-monitor/api/server.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const { hemi, hemiSepolia } = require('hemi-viem')
+const express = require('not-express')
+const pMemoize = require('promise-mem')
+
+const { getBtcVaultsData } = require('./src/btc-vaults')
+
+const cacheMinutesStr = process.env.CACHE_MINUTES || '5'
+const chain = process.env.CHAIN || 'mainnet'
+const originsStr = process.env.ORIGINS || `http://localhost:3000`
+const portStr = process.env.PORT || '3003'
+
+const chainId = chain === 'mainnet' ? hemi.id : hemiSepolia.id
+const maxAge = Number.parseInt(cacheMinutesStr) * 60 * 1000
+const limitedGetBtcVaultsData = pMemoize(getBtcVaultsData, { maxAge })
+
+const app = express()
+app.use(express.cors({ origin: originsStr.split(',') }))
+
+app.get('/', async function (req, res) {
+  const vaultsData = await limitedGetBtcVaultsData(chainId)
+  res.writeHead(200, { 'Content-Type': 'application/json' })
+  res.end(JSON.stringify(vaultsData))
+})
+
+app.listen(Number.parseInt(portStr))
```

Comment:
I would add a log info message after this
```suggestion
app.listen(Number.parseInt(portStr))

// eslint-disable-next-line no-console
console.info(`API server listening on port ${port}`)
```


it has happened to me a few times that I run the server locally, and as nothing happens (as there is no logs), I thought that something hang or was broken because I expected some output in the terminal 😅